adapters/v1/backend.go

```diff
@@ -122,11 +124,9 @@ func (a *BackendAdapter) SendStatus(ctx context.Context, step int) error {
 	report.ParentAction = workload.ParentJobID
 	report.Details = details[step]
 
-	ReportErrorsChan := make(chan error)
```

Comment:
@rcohencyberarmor is this change part of the access token? is it a fix?

Reply:
yes, this is part of the change of the access token, the former version of this function is backend repo needed channel of errors, and in the newer versions it is not needed